packages/graphql-playground-react/src/components/PlaygroundWrapper.tsx

```diff
@@ -379,7 +387,9 @@ class PlaygroundWrapper extends React.Component<
                 fixedEndpoints={Boolean(this.state.configString)}
                 headers={this.state.headers}
                 configPath={this.props.configPath}
-                workspaceName={this.state.activeProjectName}
+                workspaceName={
+                  this.state.activeProjectName || this.props.workspaceName
```

Comment:
Please change this to the other way around - `this.props.workspaceName || ...` as otherwise it's not possible to overwrite the workspaceName when using a graphql config